### PR TITLE
Migrate spec file from hangouts to googlechat

### PR DESCRIPTION
**Pull request by praiskup**

### CODE REVIEW

File: purple-googlechat.spec [hunk 1]
```diff
@@ -12,8 +12,8 @@ Epoch: 1
 Summary: Hangouts plugin for libpurple
```
Comment (EionRobb) [inline purple-googlechat.spec:12]: Might need a summary change too?

Comment (praiskup) [reply purple-googlechat.spec:12]: indeed

Comment (EionRobb) [reply purple-googlechat.spec:12]: and a few other places in there too :)

**Comment by praiskup** [comment]
ptal

**Comment by EionRobb** [comment]
Thanks for changing this up! :)

**Pull request merged**